hamlpy/test/test_compiler.py

```diff
@@ -221,10 +221,19 @@ def test_attr_wrapper(self):
   </body>
 </html>
 <script type="text/javascript">
-// <![CDATA[
-// ]]>
+  //<![CDATA[
+\x20\x20\x20\x20
```

Comment:
These are just spaces - need to use escapes because otherwise PyCharm removes trailing whitespace